deploy/updater/simulator.py

```
@@ -1,152 +1,301 @@
+    data = []
+    for district_name, types in alert_items:
+        d = D[district_name]
+        data.append(make_region_record(d["regionId"], "District", d["regionName"], d["regionEngName"], types))
+    return data
+
+
+def build_etryvoga_data(notification_items, base_id: int):
+    """Побудувати список для alerts:etryvoga:full:data.
+
+    Формат: [{"regionId": "...", "type": "DRONE", "id": <int>}, ...]
+    id починається з base_id+1 і зростає на 1 для кожного запису.
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `і` (CYRILLIC SMALL LETTER BYELORUSSIAN-UKRAINIAN I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF002)